numpy/random/mtrand/mtrand.pyx

```diff
@@ -713,10 +735,11 @@ cdef class RandomState:
 
         Parameters
         ----------
-        state : tuple(str, ndarray of 624 uints, int, int, float)
+        state : tuple(tuple(int, str), ndarray of 624 uints, int, int, float)
             The `state` tuple has the following items:
 
-            1. the string 'MT19937', specifying the Mersenne Twister algorithm.
+            1. the version of the RNG methods, and the string 'MT19937',
+               specifying the Mersenne Twister algorithm.
```

Comment:
Any reason in particular that you packed the version and `"MT19937"` into a single tuple? It seems weird to me -- I'd expect it either to go as its own entry at the beginning or the end.

Or I guess if we wanted to get fancy then it might make sense to pack the internal RNG state entries into a single entry so we're prepared for later supporting other backends.

But OTOH I'm not sure exactly which data is MT19937 specific here, and [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it), so I'd rather just make it the first entry in the tuple.